chameleon/src/test/java/org/hibernate/omm/query/NativeQueryTests.java

```diff
@@ -28,7 +28,10 @@ void testNativeQueryWithoutParameter() {
             return book;
         });
 
-        var nativeQuery = "{ find: \"books\", filter: { _id: { $eq: ? } }, projection: { _id: 1, author: 1, publishYear: 1, title: 1 } }";
+        var nativeQuery = "{ aggregate: \"books\", pipeline: [" +
```

Comment:
To simplify things, let's only support aggregate for native queries, and require a $project stage, at least to start with.

Reply:
As I explained previously, relying too much on $project is dangerous. If our library only supports single entity CRUD, it might be fine; but even in that case, it involves many subtleness. Like the special logic for `_id` (implicitly included if not explicitly specified), and negative project spec (`fieldName: 0`).